Retrie/Run.hs

```diff
@@ -48,15 +48,16 @@ import Retrie.Util
 -- >   return $ apply rr
 --
 -- To run the script, compile the program and execute it.
-runScript :: (Options -> IO (Retrie ())) -> IO ()
-runScript f = runScriptWithModifiedOptions (\opts -> (opts,) <$> f opts)
+runScript :: FixityEnv -> (Options -> IO (Retrie ())) -> IO ()
```

Comment:
This is my only real hangup with this PR. I don't love requiring a `FixityEnv` to either of these run functons.

Instead, you could change the `def` to `mempty`, and people can use the `f` callback to set a custom fixityEnv on Options.

Reply:
I have fixed this and rolled back `defaultOptions` to its previous type signature. Let me know if this is not what you had in mind